clang/test/CodeGen/ubsan-type-ignorelist.cpp

```diff
@@ -1,23 +0,0 @@
-// Verify ubsan vptr does not check down-casts on ignorelisted types.
-// RUN: echo "type:_ZTI3Foo" > %t-type.ignorelist
-// RUN: %clang_cc1 -triple x86_64-linux-gnu -fsanitize=vptr -fsanitize-recover=vptr -emit-llvm %s -o - | FileCheck %s --check-prefix=DEFAULT
```

Comment:
I'd keep this test untached.

Reply:
Yep, fixed in https://github.com/llvm/llvm-project/pull/107332/commits/8e473070c9045395d627bfece040e7d7985c0943

Thanks